src/lib/config/common.js

```diff
@@ -8,10 +8,10 @@ export const DEFAULT_LANGUAGE = 'ENG';
 // text key is used for semantic ui dropdown options display
 
 export const ACQ_ORDER_STATUSES = [
-  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled' },
-  { value: 'RECEIVED', text: 'Received', label: 'Received' },
-  { value: 'ORDERED', text: 'Ordered', label: 'Ordered' },
   { value: 'PENDING', text: 'Pending', label: 'Pending' },
+  { value: 'ORDERED', text: 'Ordered', label: 'Ordered' },
+  { value: 'RECEIVED', text: 'Received', label: 'Received' },
+  { value: 'CANCELLED', text: 'Cancelled', label: 'Cancelled' },
```

Comment:
Looking at the code, I don't think there would be any problem using order as an attribute, but I don't see any benefit either.

Reply:
IMO for the code maintenance reasons it should either have the order attribute (more explicitly telling us as developers that the order matters for the users).
Minimum would be adding a comment which tells us to preserve the ordering.